docs/user-guide/replacing-providers.ipynb

```diff
@@ -0,0 +1,277 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "86de99b0-3170-45d6-84eb-adbd622af936",
+   "metadata": {},
+   "source": [
+    "# Replacing providers\n",
```

Comment:
This is not exactly what the issue was about. I think `insert` and `__setitem__` can be found in the API docs. The issue was about documenting how to *continue from intermediate results*. Someone who wants to do that might not look for "replacing providers" (if they would, then they would already know how to do that). I think we should make this clear in the title, and maybe stick to the first example on this page (linking the `insert` can be useful).

If you want to keep the "replacing provider" example, I'd suggest to make this into another recipe?

Reply:
Sounds good!